client/src/components/DatasetContainer/tests/__snapshots__/datasetContainer.test.tsx.snap

```diff
@@ -11,7 +11,9 @@ exports[`rendering of the DatasetContainer checks if various text fields are vis
       <div
         class="grid-container-desktop-lg"
       >
-        <div>
+        <div
+          class="undefined undefined"
```

Comment:
This seems like a bug somewhere? Or is it just an artifact of the unit test?

Reply:
yeah seems like an artifact of modules, https://github.com/usds/justice40-tool/issues/513